app/src/main/java/com/example/evently/ui/organizer/EventPeopleFragment.java

```diff
@@ -50,7 +48,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
         // Must use parent fragment manager so that the children tabs will have access to
         // viewmodel...
-        viewPager.setAdapter(new EventPeopleAdapter(getParentFragmentManager(), getLifecycle()));
+        // Hierarchy: EventDetailsFragment -> EventPeopleFragment -> EntrantsFragment
+        viewPager.setAdapter(new EventPeopleAdapter(requireParentFragment()));
```

Comment:
Huh, surprised this change still allows `EntrantsFragment` to access the viewmodel from `EventDetailsFragment` - I thought this constructor just uses the fragment manager from `EventPeopleFragment`